skyportal/handlers/api/followup_request.py

```diff
@@ -194,10 +196,10 @@ def put(self, assignment_id):
         data['id'] = assignment_id
         data["requester_id"] = self.associated_user_object.id
 
-        modok = (
-            "System admin" in [a.id for a in self.current_user.acls]
-            or assignment.requester.username == self.current_user.username
-        )
+        modok = assignment.run.group_id in [
```

Comment:
This should be reverted back. Only a system admin or the assignment creator should be able to modify an assignment

Reply:
done